telluric/vectors.py

```diff
@@ -14,11 +13,21 @@
 from mercantile import Bbox, xy_bounds, tiles
 
 from rasterio.crs import CRS
-from typing import Tuple, Iterator, List
+from typing import Tuple, Iterator
 
 from telluric.constants import DEFAULT_CRS, EQUAL_AREA_CRS, WGS84_CRS, WEB_MERCATOR_CRS
-from telluric.plotting import NotebookPlottingMixin
 from telluric.util.projections import transform
+try:
+    from telluric.plotting import NotebookPlottingMixin
+except ImportError:
+    warnings.warn(
+        "Visualization dependencies not available, plotting will not work",
+        ImportWarning,
+        stacklevel=2,
+    )
+
+    class NotebookPlottingMixin:  # type: ignore
+        pass
```

Comment:
You're totally right, I found an alternative approach, please take a look. For tile server I still retained the warning because I want to give a different message.